Ta2-beautyandthebeast Pull Request updated for 1 file upload

breese5 opened the pull request:
Bryant & Vishnu assignment 2 submission

brinnaebent committed:
Update README.md

Commit:
Added python version

brinnaebent committed:
update requirements

rajivsraman commented:
The code runs well - I was able to checkout your branch in Terminal and run the Python script successfully. Your output is cleanly formatted, and it was helpful to see confirmation messages that the update/delete sections were done.

brinnaebent commented:
Please revert changes on README and requirements.txt. Should only have 1 file changed. @breese5

breese5 committed:
Recommit for grading

per Daniel's request reset the commit to Dr bent's readME/requirements and now committing our assignment file again.
